app/components/helpers/HelpButton.tsx

```diff
@@ -66,7 +67,25 @@ const HelpButton: React.FunctionComponent<Props> = ({
               </li>
             </ul>
           ) : (
-            ''
+            <ul>
+              <li>
+                <span role="img" aria-label="open book emoji">
+                  📖
+                </span>
+                <a href={docsUrl} target="_blank" rel="noopener noreferrer">
+                  See the help documentation
+                </a>
+              </li>
+              <li>OR</li>
+              <li>
+                <span role="img" aria-label="letter emoji">
+                  ✉️
```

Comment:
Using font awesome icons would ensure that we have the same icon for all browsers/OS. This looks like a yellow rectangle to me (chromium/archlinux)

Reply:
Yeah I was wondering if I should switch from using the unicode icons here in both views. Fontawesome would look a lot flatter though as it's only one colour etc. Partly why I was wondering if they were custom icons by Raj before.

Reply:
<img width="758" alt="Screenshot 2021-04-26 at 12 55 43 PM" src="https://user-images.githubusercontent.com/35253/116142461-c96f1800-a68e-11eb-845a-333cacbfe008.png">
 There's another unicode icon at least in the analyst dashboard as well.

Reply:
![image](https://user-images.githubusercontent.com/2684022/116142985-6762e280-a68f-11eb-8c27-59425e4bfa05.png)

This is what those look like on my machine. 
Maybe if we used a specific font those would be rendered the same way on all browsers. Not sure if BC Sans, which is the one we're supposed to use according to the [BC design system](https://developer.gov.bc.ca/Design-System/About-the-Design-System), has unicode characters in it.

Reply:
Yeah we can try setting a specific font for unicode icons and then fallback to fontawesome if they still... don't look awesome 😬

Reply:
@matthieu-foucault it's probably a problem of the specific emoji font you have installed, since ArchLinux leaves you more on your own to do many of the things we take for granted on other OS's: 

> Emojis should work without any configuration once you have at least one emoji font installed of supported format.
https://wiki.archlinux.org/index.php/fonts#Emoji_and_symbols

Emojis are rarely an issue on Mac and Windows these days, which are the OS's we expect the vast majority of users to be on. As a result many are now leaving behind SVG libs like FontAwesome to use the natively installed unicode icons. 

For the occasional character that's not rendered (say they're using ArchLinux), it simply underscores the importance of not using icons alone to convey meaning - in our case, the text link is well described and the icon is just decoration.

Reply:
Yeah I haven't tried to use unicode symbols in years for a UI. Should we stick with what we have now then since they work alright on Windows/Mac?

Reply:
Yes, it should be fine to stick with the unicode. We have teammates using Windows machines to help us QA them, and we can also use references like [Emojipedia](https://emojipedia.org/envelope/) which helpfully lists the cross-platform support/rendering of a specific character (in this case, scrolling down you can see a snapshot of how the envelope char that we're adding renders in MS products).

Reply:
FYI, it looks like I see the Windows emojis on CIIP, but the Google ones on Github, so the page's font does seem to have an impact on which emojis are rendered (at least for me). Somehow all of the Microsoft ones have a thick border, which is an odd design choice. The google envelope is the yellow rectangle one (with a slightly different yellow for the lines): https://emojipedia.org/envelope/

Reply:
> I see the Windows emojis on CIIP, but the Google ones on Github, so the page's font does seem to have an impact on which emojis are rendered (at least for me)

Interestingly, since Github gets the interesting challenge of trying to render emojis on *goodness knows what arcane operating system* its users may be on, inspection shows they're using a clever progressive enhancement technique using web components / custom HTML elements to render a `.png` fallback image if unavailable:

<img width="755" alt="Screen Shot 2021-04-26 at 2 47 25 PM" src="https://user-images.githubusercontent.com/5522075/116155111-84eb7880-a69e-11eb-9914-303f3ec6eb63.png">